projects/plugins/jetpack/extensions/blocks/ai-assistant/hooks/use-suggestions-from-openai/index.js

```diff
@@ -14,7 +14,8 @@ import { buildPromptForBlock } from '../../lib/prompt';
 import { askJetpack, askQuestion } from '../../lib/suggestions';
 import { getContentFromBlocks, getPartialContentToBlock } from '../../lib/utils/block-content';
 
-const debug = debugFactory( 'jetpack-ai-assistant:event:fullMessage' );
+const debug = debugFactory( 'jetpack-ai-assistant:event' );
+const debugPromt = debugFactory( 'jetpack-ai-assistant:prompt' );
```

Comment:
```suggestion
const debugPrompt = debugFactory( 'jetpack-ai-assistant:prompt' );
```
Small typo, would need an update on all calls